docs/index.md

```diff
@@ -1,118 +1,142 @@
+    ```
 
-- Elasticsearch
-- MongoDB
+    - Activate the virtual environment:
+    ```bash
+    source venv/bin/activate
+    ```
 
-## CLI key concepts
+If you only need to integrate [learning statement models](./models/index.md) feature in your project, you don't need to
+install the `backends`, `cli` or `lrs` extra dependencies, the **core library** is what you need:
 
-At first, Ralph has been designed as a CLI to batch process your logs using
-base commands and UNIX standard streams (`stdin`, `stdout`) to connect them in
-a pipeline. A base example pipeline may be:
-
-```sh
-$ ralph read --backend swift my_archive.gzip | \
-    gunzip | \
-    ralph write --backend es
+```bash
+pip install ralph-malph
 ```
 
-In this small pipeline, we stream `my_archive.gzip` content from a Swift
-container to the standard output (using the `read` command), uncompress the
-content (using the `gunzip` command), and bulk insert logs in an ElasticSearch
-index (using the `write` command).
+If you want to use the [Ralph LRS server](./api/index.md), add the `lrs` flavour in your installation. What's more, you have to choose the type of backend you will use for LRS data storage (`backend-clickhouse`,`backend-es`,`backend-mongo`).
 
-As UNIX is beautiful, Ralph offers many powerful possibilities by combining its
-commands with other standard commands or command line tools.
+- Install the **core package** with the LRS and the Elasticsearch backend for example:
 
-## Quick start guide
+=== "bash"
 
-Ralph is distributed as a [python
-package](https://pypi.org/project/ralph-malph/) and a [Docker
-image](https://hub.docker.com/repository/docker/fundocker/ralph).
+    ```bash
+    pip install ralph-malph[backend-es,lrs]
+    ```
 
-> If you choose to install `ralph` in your native environment (without using
-> Docker), please make sure that **Python 3.11** is installed (and your default
-> python distribution).
+=== "zsh"
 
-Ralph package can be installed from PyPI using the `pip` tool:
+    ```zsh
+    pip install "ralph-malph[backend-es,lrs]"
+    ```
 
-```sh
-# Create a new virtualenv (optional)
-$ python3.11 -m venv venv
-$ source venv/bin/activate
+- Add the `cli` flavour if you want to use the LRS on the command line: 
 
-# Install the full package (in a virtualenv)
-(venv) $ pip install \
-    ralph-malph[backend-es,backend-ldp,backend-lrs,backend-mongo,backend-swift,backend-ws,cli,lrs]
+=== "bash"
 
-# Install only the core package with the Elasticsearch backend and the LRS (in
-# a virtualenv)
-(venv) $ pip install ralph-malph[backend-es,lrs]
+    ```bash
+    pip install ralph-malph[backend-es,lrs,cli]
+    ```
 
-# Test the ralph command (it should be in your PATH)
-(venv) $ ralph --help
-```
+=== "zsh"
 
-If you only need to integrate `ralph` models in your project, you don't need to
-install the `backends`, `cli` or `lrs` extra dependencies, the core package is
-what you need:
+    ```zsh
+    pip install "ralph-malph[backend-es,lrs,cli]"
+    ```
```

Comment:
```suggestion
    ```zsh
    pip install "ralph-malph[backend-es,lrs,cli]"
    ```
```
This is a bit too detailed in my opinion.

Reply:
Ok, I'll remove it